inc/admin.php

```diff
@@ -2069,4 +2074,53 @@ public function survey_category( $value, $scale = 1 ) {
 
 		return 0;
 	}
+
+	/**
+	 * Determines whether the exceed quota warning should be displayed to users.
+	 *
+	 * This function checks if the user's quota usage has exceeded a predefined limit
+	 * and returns a boolean value indicating whether the warning should be shown.
+	 *
+	 * @return bool True if the exceed quota warning should be displayed, false otherwise.
+	 */
+	public function should_show_exceed_quota_warning() {
+		$current_screen = get_current_screen();
+		if ( ( defined( 'DOING_AJAX' ) && DOING_AJAX ) ||
+			is_network_admin() ||
+			! current_user_can( 'manage_options' ) ||
+			! $this->settings->is_connected() ||
+			empty( $current_screen )
+		) {
+			return false;
+		}
+		if ( get_option( 'optml_notice_hide_upg', 'no' ) === 'yes' ) {
+			return false;
+		}
+
+		if ( ! str_contains( $current_screen->base, 'page_optimole' ) ) {
+			return false;
+		}
```

Comment:
Same thing. I doubt we need the check here.

Reply:
I have removed all additional condition checks.